uqcsbot/uptime.py

```diff
@@ -18,27 +18,38 @@ def __init__(self, bot: UQCSBot):
 
     @commands.Cog.listener()
     async def on_ready(self):
-        channel = discord.utils.get(self.bot.get_all_channels(), name=self.CHANNEL_NAME)
-
-        if channel != None:
-            await channel.send("I have rebooted!")
+        channel = discord.utils.get(
+            self.bot.uqcs_server.channels, name=self.CHANNEL_NAME
+        )
+
+        if channel is not None:
+            if random.randint(1, 100) == 1:
+                await channel.sent("Oopsie, I webooted uwu >_<")
```

Comment:
Really... Fine

Reply:
hey, you weren't supposed to notice that